lib/pull.ts

```diff
@@ -286,7 +286,7 @@ function cleanOutputFiles() {
 
     if (
       item.isFile() &&
-      /\.js(on)?|\.xml|\.strings(dict)?$|\.swift$/.test(item.name)
+      /\.js(on)?|\.ts|\.xml|\.strings(dict)?$|\.swift$/.test(item.name)
```

Comment:
Small pre-existing bug where we weren't clearing out the `index.d.ts` file between `pull`s.